integration/tests/karpenter/karpenter_test.go

```diff
@@ -0,0 +1,83 @@
+package karpenter
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/weaveworks/eksctl/integration/runner"
+	"github.com/weaveworks/eksctl/integration/tests"
+	clusterutils "github.com/weaveworks/eksctl/integration/utilities/cluster"
+	"github.com/weaveworks/eksctl/integration/utilities/kube"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/karpenter"
+	"github.com/weaveworks/eksctl/pkg/testutils"
+)
+
+var params *tests.Params
+
+func init() {
+	testing.Init()
+	if err := api.Register(); err != nil {
+		panic(fmt.Errorf("unexpected error registering API scheme: %w", err))
+	}
+	params = tests.NewParams("")
```

Comment:
This needs to be given a prefix if you're planning to use the cluster name generated by `tests.NewParams`.

Reply:
I'm not planning on that.